### PR TITLE
chore: remove unused operations code

### PR DESCRIPTION
**Pull request by chandumlg**
# Description

Cleaned up the following code:
1. Unused operations code
2. Legacy app handlers

## Notion Ticket

https://www.notion.so/rudderstacks/8aac9087df644365acdf64e28e290153?v=6e06b0a5ade24f0aa5ffe05dc2972e84&p=03d78048dd5448cf801e237d15f7859d

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: services/sql-migrator/migrations_vfsdata.go [hunk 1]
```diff
@@ -369,6 +388,7 @@ var MigrationAssets = func() http.FileSystem {
 		fs["/warehouse"].(os.FileInfo),
 	}
 	fs["/jobsdb"].(*vfsgen۰DirInfo).entries = []os.FileInfo{
+		fs["/jobsdb/.DS_Store"].(os.FileInfo),
```
Comment (lvrach) [inline services/sql-migrator/migrations_vfsdata.go:391]: `.DS_Store` should not be included here.

**Review by lvrach — APPROVED** [review]

**Review by fracasula — APPROVED** [review]

**Comment by chandumlg** [comment]
Will address the clean up comments in the next pr.

File: services/sql-migrator/migrations_vfsdata.go [hunk 2]
```diff
@@ -21,340 +21,359 @@ var MigrationAssets = func() http.FileSystem {
 	fs := vfsgen۰FS{
```
Comment (atzoum) [inline services/sql-migrator/migrations_vfsdata.go:21]: is there a quick explanation for what this piece of code does?

**Review by atzoum — COMMENTED** [review]
@chandumlg I am not the appropriate person to approve this, nevertheless I feel the need to express my gratitude for deleting all various `LegacyStart`! 👍

**Pull request merged**